graphstorm-processing/graphstorm_processing/constants.py

```diff
@@ -44,6 +46,8 @@
 ################# Numerical transformations  ################
 VALID_IMPUTERS = ["none", "mean", "median", "most_frequent"]
 VALID_NORMALIZERS = ["none", "min-max", "standard", "rank-gauss"]
+VALID_OUTDTYPE = ["float32", "float64"]
```

Comment:
Maybe we can make the names of types constants as well?

```
TYPE_FLOAT32 = "float32"
TYPE_FLOAT64 = "float64"
VALID_OUTDTYPE = [TYPE_FLOAT32, TYPE_FLOAT64]
```

The change all raw string refs to the constants.